packages/lodestar/src/util/httpClient.ts

```diff
@@ -1,30 +1,41 @@
 import Axios, {AxiosResponse, AxiosInstance} from "axios";
+import { ILogger } from "../logger";
+
+export interface HttpClientOptions {
+  // Add more options if needed
+  urlPrefix?: string;
+}
 
 export class HttpClient {
-  private _client: AxiosInstance;
+  private client: AxiosInstance;
+  private logger: ILogger;
 
-  public constructor(urlPrefix?: string) {
-    const _urlPrefix = urlPrefix? urlPrefix : "";
-    this._client = Axios.create({
-      baseURL: _urlPrefix
+  public constructor(opt: HttpClientOptions, logger: ILogger) {
```

Comment:
to follow convention, it should be 
`public constructor(opt: HttpClientOptions, {logger}: {logger: ILogger})`